src/components/EligibilityForm/components/ContactInformation.tsx

```diff
@@ -24,34 +26,65 @@ const InputWraper = styled(Box)`
   && {
     margin-bottom: 0;
   }
+
+  .heatingEnergyContactInformations .fr-fieldset__legend {
+    font-weight: bold !important;
+    padding-left: 0px;
+    margin-left: 0px;
+  }
 `;
 
 export const fieldLabelInformation = {
   structure: {
-    label: 'Type de bâtiment :',
+    label: 'Vous êtes...',
     inputs: [
-      { value: 'Copropriété', label: 'Copropriété', id: 'copropriete' },
-      { value: 'Tertiaire', label: 'Tertiaire', id: 'tertiaire' },
+      { value: 'Copropriété', label: 'Copropriétaire', id: 'copropriete' },
       {
         value: 'Maison individuelle',
-        label: 'Maison individuelle',
+        label: 'Propriétaire de maison individuelle',
         id: 'maison',
       },
+      { value: 'Tertiaire', label: 'Professionnel', id: 'tertiaire' },
+    ],
+  },
+  companyTitle: 'Votre structure',
+  company: 'Nom de votre structure',
+  companyType: {
+    label: 'Type de structure',
+    inputs: [
+      { value: 'Syndic de copropriété', label: 'Syndic de copropriété', id: 'syndic' },
       { value: 'Bailleur social', label: 'Bailleur social', id: 'bailleur' },
+      { value: 'Gestionnaire de parc tertiaire', label: 'Gestionnaire de parc tertiaire', id: 'gestionnaire' },
+      { value: "Bureau d'études ou AMO", label: "Bureau d'études ou AMO", id: 'bureau' },
+      { value: 'Mandataire / délégataire CEE', label: 'Mandataire / délégataire CEE', id: 'mandataire' },
+      { value: 'Autre', label: 'Autre', id: 'autre' },
     ],
   },
-  lastName: 'Nom :',
-  firstName: 'Prénom :',
-  company: 'Établissement',
-  email: 'Email :',
-  phone: 'Téléphone :',
+  contactDetailsTitle: 'Vos coordonnées',
+  lastName: 'Nom',
+  firstName: 'Prénom',
+  email: 'Email',
+  phone: 'Téléphone (optionnel)',
+  nbLogements: 'Nombre de logements',
+  demandCompanyType: {
+    label: 'Votre demande concerne',
+    inputs: [
+      { value: 'Copropriété', label: 'une copropriété', id: 'copro' },
+      { value: 'Maison individuelle', label: 'une maison individuelle', id: 'maison' },
+      { value: 'Bâtiment tertiaire', label: 'un bâtiment tertiaire', id: 'batiment' },
+      { value: 'Bailleur social', label: 'un bailleur social', id: 'bailleur' },
+      { value: 'Autre', label: 'autre', id: 'autre' },
+    ],
+  },
+  demandCompanyName: 'Nom de la structure accompagnée',
+  demandArea: 'Surface en m2',
   heatingEnergy: {
-    label: 'Mode de chauffage actuel :',
+    label: 'Mode de chauffage',
     inputs: [
       { value: 'électricité', label: 'Électricité', id: 'electricite' },
       { value: 'gaz', label: 'Gaz', id: 'gaz' },
       { value: 'fioul', label: 'Fioul', id: 'fioul' },
-      { value: 'autre', label: 'Autre / Je ne sais pas', id: 'autre' },
+      { value: 'autre', label: 'Autres', id: 'autre' },
```

Comment:
```suggestion
      { value: 'autre', label: 'Autre', id: 'autre' },
```
Mode de chauffage donc autre au singulier plutôt

Reply:
Je demande à Florence